backend/package-lock.json

```diff
@@ -220,6 +220,14 @@
       "resolved": "https://registry.npmjs.org/@sideway/pinpoint/-/pinpoint-2.0.0.tgz",
       "integrity": "sha512-RNiOoTPkptFtSVzQevY/yWtZwf/RxyVnPy/OcA9HBM3MlGDnBEYL5B41H0MTn0Uec8Hi+2qUtTfG2WWZBmMejQ=="
     },
+    "node_modules/@tootallnate/once": {
```

Comment:
this file shouldn't be changed

Reply:
Shouldnt package lock files change if we install package in shared folder and then run command npm run update:shared:all?

Reply:
packages-lock file shouldn't just just because